cmd/neofs-adm/internal/modules/storagecfg/root.go

```diff
@@ -0,0 +1,304 @@
+	if c.Wallet.Account == "" {
+		c.Wallet.Account = getString("Wallet account: ")
+	}
+
+	accH, err := flags.ParseAddress(c.Wallet.Account)
+	fatalOnErr(err)
+
+	acc := w.GetAccount(accH)
+	if acc == nil {
+		fatalOnErr(errors.New("can't find account in wallet"))
+	}
+
+	c.Wallet.Password, err = input.ReadPassword(fmt.Sprintf("Account password for %s: ", c.Wallet.Account))
+	fatalOnErr(err)
+
+	err = acc.Decrypt(c.Wallet.Password, keys.NEP2ScryptParams())
+	fatalOnErr(err)
+
+	c.AuthorizedKeys = append(c.AuthorizedKeys, hex.EncodeToString(acc.PrivateKey().PublicKey().Bytes()))
+
+	network := getString("Choose network [mainnet]/testnet: ")
+	switch network {
+	case "":
+		network = "mainnet"
+	case "testnet", "mainnet":
+	default:
+		fatalOnErr(errors.New("invalid network"))
+	}
+
+	c.MorphRPC = n3config[network].MorphRPC
+
+	depositGas(acc, network)
+
+	c.Attribute.Locode = getString("UN-LOCODE attribute in [XX YYY] format: ")
+	c.AnnouncedAddress = getString("Publicly announced address: ")
+	c.Endpoint = getString(fmt.Sprintf("Listening address [%s]: ", defaultDataEndpoint))
+	if c.Endpoint == "" {
+		c.Endpoint = defaultDataEndpoint
+	}
+	c.ControlEndpoint = getString(fmt.Sprintf("Listening address (control endpoint) [%s]: ", defaultControlEndpoint))
+	if c.ControlEndpoint == "" {
+		c.ControlEndpoint = defaultControlEndpoint
+	}
+
+	c.TLSCert = getPath("TLS Certificate (optional): ")
+	if c.TLSCert != "" {
+		c.TLSKey = getPath("TLS Key: ")
+	}
+
+	c.Relay = getConfirmation(false, "Use node as a relay? yes/[no]: ")
+	if !c.Relay {
+		p := getPath("Path to the storage directory (all available storage will be used): ")
+		c.BlobstorPath = filepath.Join(p, "blob")
+		c.MetabasePath = filepath.Join(p, "meta")
+	}
+
+	out := applyTemplate(c)
+	fatalOnErr(ioutil.WriteFile(args[0], out, 0644))
+}
+
+func getString(prompt string) string {
+	s, err := input.ReadLine(prompt)
+	fatalOnErr(err)
+	return s
+}
+
+type filenameCompleter struct{}
+
+func (filenameCompleter) Do(line []rune, pos int) (newLine [][]rune, length int) {
+	prefix := string(line[:pos])
+	dir := filepath.Dir(prefix)
+	de, err := os.ReadDir(dir)
+	if err != nil {
+		return nil, 0
+	}
+
+	for i := range de {
+		name := filepath.Join(dir, de[i].Name())
+		if strings.HasPrefix(name, prefix) {
+			tail := []rune(strings.TrimPrefix(name, prefix))
+			if de[i].IsDir() {
+				tail = append(tail, filepath.Separator)
+			}
+			newLine = append(newLine, tail)
+		}
+	}
+	if pos != 0 {
+		return newLine, pos - len([]rune(dir))
+	}
+	return newLine, 0
+}
+
+func getPath(prompt string) string {
+	readline.SetAutoComplete(filenameCompleter{})
+	p, err := readline.Line(prompt)
+	fatalOnErr(err)
+	return p
+}
+
+func getConfirmation(def bool, prompt string) bool {
+	for {
+		s, err := readline.Line(prompt)
+		fatalOnErr(err)
+
+		switch strings.ToLower(s) {
+		case "y", "yes":
+			return true
+		case "n", "no":
+			return false
+		default:
+			if len(s) == 0 {
+				return def
+			}
+			continue
```

Comment:
Maybe it's worth to print what are we waiting for. Otherwise `continue` is redundant

Reply:
It is printed in `readline.Line` above, so the workflow is like this:
```
Use node as a relay? yes/[no]: kek
Use node as a relay? yes/[no]: please
Use node as a relay? yes/[no]: yes
...
```

Reply:
Removed `continue`.